Sources/Web3Core/KeystoreManager/BIP32Keystore.swift

```diff
@@ -216,6 +216,54 @@ public class BIP32Keystore: AbstractKeystore {
         try encryptDataToStorage(password, data: serializedRootNode, aesMode: self.keystoreParams!.crypto.cipher)
     }
 
+    /// Fast generation addresses for current account
+    /// used for shows which address user will get when changed number of his wallet
+    /// - Parameters:
+    ///   - password: password of seed storage
+    ///   - number: number of wallets addresses needed to generate from 0 to number-1
+    /// - Returns: Array of addresses generated from 0 to number bound, or empty array in case of error
+    public func getAddressForAccount(password: String, number: Int) -> [EthereumAddress] {
+        guard let decryptedRootNode = try? getPrefixNodeData(password) else {
+            return []
+        }
+        guard let rootNode = HDNode(decryptedRootNode) else {
+            return []
+        }
+        let prefixPath = self.rootPrefix
+        var pathAppendix: String?
+
+        return [Int](0..<number).compactMap({ number in
+            pathAppendix = nil
+            let path = prefixPath + "/\(number)"
+            if path.hasPrefix(prefixPath) {
+                let upperIndex = (path.range(of: prefixPath)?.upperBound)!
+                if upperIndex < path.endIndex {
+                    pathAppendix = String(path[path.index(after: upperIndex)..<path.endIndex])
+                } else {
+                    return nil
+                }
+
+                guard pathAppendix != nil else {
+                    return nil
+                }
+                if pathAppendix!.hasPrefix("/") {
+                    pathAppendix = pathAppendix?.trimmingCharacters(in: CharacterSet.init(charactersIn: "/"))
```

Comment:
please use proper swift init syntax

Reply:
createNewCustomChildAccount

Reply:
All `CharacterSet.init` were replaced with `.init` to reduce the length of lines where it occurs.